src/pages/feed/GeneralFeed/GeneralFeed.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '~@santiment-network/ui/mixins';
+
+.title {
+  @include text('body-1');
+
+  display: flex;
+  color: $rhino;
+  margin-bottom: 26px;
+
+  @include responsive('phone', 'phone-xs') {
+    margin-bottom: 16px;
+  }
+}
+
+.helpTrigger {
+  margin-left: 8px !important;
+}
+
+.scrollable {
+  margin-right: 24px;
+  overflow-y: auto;
+  max-height: 100%;
+  position: relative;
+  height: 100%;
+
+  @include responsive("desktop") {
+    width: 715px;
+  }
+
+  @include responsive("phone", "phone-xs") {
+    padding: 0;
+    margin: 0;
+  }
+}
+
+.activitiesWrapper {
+  padding-right: 32px;
+  margin: 0 !important;
+
+  @include responsive('phone', 'phone-xs') {
+    padding-right: 0;
+  }
+}
+
+.activityItem {
+  padding: 16px;
+  background: var(--white);
+  border: 1px solid var(--porcelain);
+  box-sizing: border-box;
+  box-shadow: 0 2px 8px rgba(59, 84, 106, 0.03), 0 1px 2px rgba(59, 84, 106, 0.03);
+  border-radius: $border-radius;
+  margin-top: 16px;
+}
+
+.firstActivity {
+  margin-top: 0;
+}
+
+.activityCustom {
+  margin: 0 0 16px 0 !important;
+}
+
+.activityMarkdown {
+  p {
+    // GarageInc: becuase markdown from server
+    margin-bottom: 0 !important;
+  }
```

Comment:
Could we also set our color for `p` and links inside it here? 
![image](https://user-images.githubusercontent.com/24521041/70319754-1b15c900-1834-11ea-9412-4d3b9a6f4368.png)